e2e_test/test_smoketest.sh

```diff
@@ -6,4 +6,4 @@ prepare_node
 
 (cd debug-fee-currency && forge build .)
 export COMPILED_TEST_CONTRACT=../debug-fee-currency/out/DebugFeeCurrency.sol/DebugFeeCurrency.json
-cd js-tests && ./node_modules/mocha/bin/mocha.js test_viem_smoketest.mjs --exit
+cd js-tests && ./node_modules/mocha/bin/mocha.js test_viem_smoketest.mjs --timeout 16000 --exit 
```

Comment:
Here's a useless trailing space
```suggestion
cd js-tests && ./node_modules/mocha/bin/mocha.js test_viem_smoketest.mjs --timeout 16000 --exit
```

Reply:
What tools did you use to spot that?

Reply:
Just the github diff view.